### PR TITLE
Feature branch to master Pull Request - Using Bootstrap project

### PR DESCRIPTION
**Pull request by abouhid**
This is the pull request needed for the project to be approved. Thank you!

### CODE REVIEW

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: [Changes Required]

#### Hey Team! The project looks good, but there are still a few details that we need to improve to get this project approved.

### Navbar

| Yours | Original |
|:---:|:---:|
| image | image |

- [x] Add the info about the city above the date.
-[optional] Try to add that little red detail by the end of the let part of the navbar. You can use css pseudo-elements.

- [x] The navbar needs to stay fixed when you scroll down the page. Please do that as well in your clone. 

- [x] Some elements in the navbar are creating a second row in the mobile version. Kindly fix that.
image

### Main section

- [x] In the mobile version, sometimes the blue background shows up between sections. 

| Original | Yours |
|:---:|:---:|
| image | image |

### More Stories

- [ ] The text should wrap below the images when the content is big enough.

| Yours | Original |
|:---:|:---:| 
| image  | image  |

### Editor's pick

- [ ] Please re-organize the rows and columns for the large screen sizes.

| Yours | Original |
|:---:|:---:|
| image  | image |

- [x] You are missing a few sections still: US - World - Business - Tech and Science - Culture and sports

image

### Subscribe section

- [x] The colors of the buttons is wrong

| Yours | Original |
|:---:|:---:|
| image | image |

- [x] In the newsletter section, some elements overflow in your design in the mobile screen sizes.

| Yours | Original |
|:---:|:---:|
| image | image |

### General

- [ ] Please check the padding of the page, font sizes (and when they are bold or not).

#### This is a tough project, and your clone looks really good already. With these implementations above will look even better. You really displayed  knowledge on Bootstrap and the code looks quite good :clap:
### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by abouhid** [comment]
Thanks for the review, however, I have some things to point out:
1- fixed the navbar issues
2- Added the section separations
3- On my pc here I tried to replicate what was shown in the print screen, but the original website here does not get organized that way. Therefore I am not able to clone what I can not compare my project to.
4- About the colors, I did not clone exactly the same colors on purpose, because the project focuses on applying bootstrap, hence I tried to use css as least as possible. Are you sure I should change all of that?
5- Overlapping fixed

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required

#### I understand that it shows different on your screen. So these sections are ok to leave like it is now. Can you submit a print screen for the next review? 

But there is still one thing that you did not change:

- [ ] screencapture-github-com-abouhid-Using-Bootstrap-pull-3-1594413844496 (1)

And now the fixed navbar is collapsing on mobile screen size. Kindly fix that.

- [ ] screencapture-127-0-0-1-5500-index-html-1594414383212 (1)


As for the color, you can leave as that if it is a conscient choice. There is one optional change that I will suggest to improve your project even more:
- The height of the fixed navbar could be smaller on the mobile screen size. 

#### As I stated in my first review, the project does look great. You showed a great level of technical skills and attention to detail. 

## Submit the project for a new review once you are ready.

Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by abouhid** [comment]
Oh sorry, I skipped the text/image issue. It is fixed now.
Collapsing navbar fixed
The Editor's Pick was indeed without the col-lg-3 class, fixed it now
Here the printscreen of what appears to me:
print1
print2

Hope everything is correct now for me to move on! Thanks again for the attention!

File: css/style.css [hunk 1]
```diff
@@ -128,6 +128,44 @@ form{
...
+    
+}
+
+.editor a{
+  color:black;
+}
+
+.editor a:hover{
+  color:black;
+
```
Comment (bertil291utn) [inline css/style.css]: ### Using the Same Rule Repetitively

Use a class instead:
````
.black {color: #000;}

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🔴  :shipit: ? Not yet

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

Your work is amazing you master it GitHub Flow you have a professional README file. (Check optional [changes](##Optional_changes))

#### Mandatory changes

##### Layout


- [ ] Complete the missing section. You're missing **New Section picks **section below Editors Pick section

image

 ##### CSS

- [ ] **Using the Same Rule Repetitively**, check inline comment[1]

#### Optional changes

[TAKE IN MIND]

- [ ] Don't forget to use meaningful in all your commits messages, to next projects

  - Next time you might try using this template:


  ```
  An unmeaningful title
  - Descriptive messages
  - More meaningful messages
  ```

image

- [ ] Don't forget use a CSS Table of Contents[1]

  Consider this example below:

  ```
  /* Project Meta */
  
  /* reset  */
  
  /* general */
  
  /* typography */
  
  /* grid */
  
  /* header */
  
  /* footer */
  
  /* Page template A */
  
  /* Page template B */
  
  /* Media Queries */
  ```



_[1] [Good and Bad CSS practices](https://speckyboy.com/good-bad-css-practices/)_



<hr>


Guys work in the requested changes above and request code for another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by abouhid** [comment]
It was explained on the past review that this section does not appear on my desktop. I changed the css to avoid repetitive css rules.

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  :shipit: ? Not yet

# Hey @abouhid , you've been doing a great job.

Thanks for your changes but still is not accomplished as the [rules ](https://speckyboy.com/good-bad-css-practices/)

#### Mandatory changes

 ##### CSS

- [ ] **Using the Same Rule Repetitively**[1]

Try to create a class called `.black-text {color: #000;}` then use in each element tag where you need. Example
```
<div class="title black-text">My title</div>
/* Another tag */
<div class="subtitle black-text">
```

_[1] [Good and Bad CSS practices](https://speckyboy.com/good-bad-css-practices/)_



<hr>


Guys work in the requested changes above and request code for another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Great job guys thank you for making the required changes by the previous TSE.

now you can merge to master your project is approved.

## Happy coding :tophat:

**Pull request merged**